README.md

```diff
@@ -1 +1,57 @@
-# Hello-word
\ No newline at end of file
+
+# Hello-word Project
+
+this is a simple project  microverse project that I did to help me to understand :
+How to Create branches with different base branches,
+ the concept of version control,
+and how linters are used in a real project.
+
+
+
+## Built With
+
+- Html
+- Css
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Author
+
+👤 **Author1**
+
+- GitHub: [Nezerwa](https://github.com/settings/profile)
```

Comment:
- [ ] Please since you are the only contributor to this project you can singularize the `Authors` section to `Author` and replace `Author1` with your name. :smile:
- [ ] Also you should replace `https://github.com/settings/profile` with the link to your GitHub profile